libyear-bundler.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables = ["libyear-bundler"]
   spec.require_paths = ["lib"]
   spec.required_ruby_version = ">= 2.1"
-  spec.add_dependency "bundler", "~> 1.14"
+  spec.add_dependency "bundler", ">= 1.14", "< 2.1"
```

Comment:
bundler is pretty darn stable. If you want to do `< 3` that'd be fine too.

Reply:
Done.